Mathlib/CategoryTheory/Closed/Cartesian.lean

```diff
@@ -130,8 +129,9 @@ set_option quotPrecheck false in
 /-- Morphisms from an exponentiable object. -/
 notation:30 B " ^^ " A:30 => (exp A).obj B
 
-@[simp, reassoc]
-theorem ev_coev : Limits.prod.map (𝟙 A) ((coev A).app B) ≫ (ev A).app (A ⨯ B) = 𝟙 (A ⨯ B) :=
+#adaptation_note /-- Removed simp attribute as simp can already prove it. -/
```

Comment:
The command `adaptation_note` has a limited scope, so that you should use a more basic `-- ...` syntax.

Reply:
I’ll just remove these notes then. Looks like some of the proofs broke when merging anyways.